antlr4/PostgreSQLParser.g4

```diff
@@ -82,7 +82,7 @@ stmt
      | unlisten_stmt
      | update_stmt
      | vacuum_stmt
-     | values_stmt)
+     | values_stmt) CLOSE_PAREN* combine_clause?
```

Comment:
Everything else looks great, but I don't think it's right to have this `combine_clause?` here because I shouldn't be able to do `UPDATE TABLE... UNION VALUES(1, 2, 3)`. We don't have a fuzzer here for testing bad input, but if we did and compared with psql we would get queries that pass this parser but fail the official psql parser. 

This is probably true for some things in this grammar, but this particular change sticks out to me.

Reply:
You're right again :) I have removed `combine_clause` from `stmt`. :+1: